plugins/serializers/registry.go

```diff
@@ -73,6 +74,8 @@ func NewSerializer(config *Config) (Serializer, error) {
 		serializer, err = NewGraphiteSerializer(config.Prefix, config.Template, config.GraphiteTagSupport)
 	case "json":
 		serializer, err = NewJsonSerializer(config.TimestampUnits)
+	case "splunkmetric":
+		serializer, err = NewSplunkmetricSerializer(config.TimestampUnits)
```

Comment:
Splunk only supports seconds. The serializer should not be using `config.TimestampUnits`.

Reply:
Splunk absolutely support sub second timestamps. From the default datetime.xml:
```xml
<define name="_utcepoch" extract="utcepoch, subsecond">
        <!-- update regex before '2017' -->
    <text><![CDATA[((?<=^|[\s#,"=\(\[\|\{])(?:1[012345]|9)\d{8}|^@[\da-fA-F]{16,24})(?:\.?(\d{1,6}))?(?![\d\(])]]></text>
</define>```

Reply:
I didn't say it doesn't. It only supports seconds as the unit. It supports more than that as the precision. Unit != precision.